src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -200,55 +205,17 @@ public enum CallOfLegends implements LogicCardInfo {
             energyCost M, M, M, M
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 70
+              my.hand.moveTo(hidden:true, my.deck)
+              shuffleDeck()
```

Comment:
If it is changing game state (adding/removing from hand/deck/pokemoncardset's cards/etc), it should probably be done `afterDamage`. This applies to many effects.